community/modules/file-system/gke-persistent-volume/main.tf

```diff
@@ -18,3 +18,71 @@ locals {
   # This label allows for billing report tracking based on module.
   labels = merge(var.labels, { ghpc_module = "gke-persistent-volume" })
 }
+
+locals {
+  split_filestore_id   = var.filestore_id != null ? split("/", var.filestore_id) : [null, null, null, null, null, null]
+  location             = local.split_filestore_id[3]
```

Comment:
I think these are more naturally written as:

```hcl
location = try(split("/", var.filestore_id)[3], null)
filestore_name = try(split("/", var.filestore_id)[5], null)
```

Reply:
Done / null is no longer valid for this PR